src/views/AnalysisModule/index.tsx

```diff
@@ -111,6 +170,29 @@ function AnalysisModule(props: AnalysisModuleProps) {
         },
     );
 
+    const [
+        ,
+        overviewResponse,
+        ,
+        ,
+    ] = useRequest<MultiResponse<AnalysisOverview>>(
+        {
+            url: `server://projects/${activeProject}/analysis-overview/`,
+            method: 'GET',
+            autoTrigger: true,
+            onFailure: (_, errorBody) =>
+                notifyOnFailure(_ts('analysis', 'analysisModule'))({ error: errorBody }),
+        },
+    );
+
+    const piechartData: AuthoringOrganizations = overviewResponse?.authoringOrganizations ?? [];
+
+    const timelineData: TimelineData = (analysisResponse?.results ?? []).map(d => ({
+        key: d.id,
+        value: new Date(d.createdAt).getTime(),
```

Comment:
Here value is of type Number but it is defined to be a string in the TimelineData interface. Similarly, createdAt is optional in AnalysisElement type and it should be handled properly.